From f9445bc778fb1ff800933ac7043df6da61c97792 Mon Sep 17 00:00:00 2001
From: Manish Pandey <quic_mapa@quicinc.com>
Date: Fri, 13 Oct 2023 19:38:59 +0530
Subject: [PATCH 10/27] QCLINUX: arm64: dts: qcom: qcm6490: disable sdhc1 for
 ufs target

Disable sdhc1 for QCM6490 for ufs boot target to avoid probe
for sdhc1 as vreg_l7b_2p9 is shared regulator for both ufs vcc
and emmc vcc. Currently this is causing probe failure for ufs.

Change-Id: I392d2c3302cbae7256d7c6f6045dc70cdde58368
Signed-off-by: Manish Pandey <quic_mapa@quicinc.com>
Signed-off-by: Salendarsingh Gaud <quic_sgaud@quicinc.com>
Signed-off-by: Atul Dhudase <quic_adhudase@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcm6490-idp.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
index 65bf5a99b5dc..8f00c8b59375 100644
--- a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
+++ b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
@@ -250,6 +250,10 @@ &gpi_dma1 {
 	status = "okay";
 };
 
+&sdhc_1 {
+	status = "disabled";
+};
+
 &pm8350c_pwm {
 	status = "okay";
 };
-- 
2.25.1

